app/models/package_manager/elm.rb

```diff
@@ -56,25 +56,16 @@ def self.versions(project, _name)
       end
     end
 
-    def self.dependencies(name, version, project)
-      find_and_map_dependencies(name, version, project)
-    end
-
-    def self.find_dependencies(name, version)
-      url = "https://raw.githubusercontent.com/#{name}/#{version}/elm-package.json"
-
-      begin
-        response = request(url)
-        if response.status == 200
-          contents = response.body
-          dependencies = Bibliothecary.analyse_file("elm-package.json", contents).first.try(:fetch, :dependencies)
-          dependencies
-        else
-          []
+    def self.dependencies(name, version, _mapped_project)
+      get("http://package.elm-lang.org/packages/#{name}/#{version}/elm.json")
```

Comment:
This was the only JSON endpoint I could find that returns the data (it's the one that's used as an XHR to fetch the data on web)